stable/testlink/templates/tls-secrets.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  labels:
+    app: "{{ template "testlink.fullname" $ }}"
+    chart: "{{ template "testlink.chart" $ }}"
+    release: {{ $.Release.Name | quote }}
+    heritage: {{ $.Release.Service | quote }}
+type: kubernetes.io/tls
+data:
+  tls.crt: {{ .certificate | b64enc }}
+  tls.key: {{ .key | b64enc }}
+---
```

Comment:
Could you remove this line?

Reply:
sure

Reply:
wordpress helm files has this line too, that's why I kept it:

https://github.com/helm/charts/blob/c330fa62be204bba0c8e2050d4935b56ff27fd04/stable/wordpress/templates/tls-secrets.yaml#L14-L18

Reply:
Yes, it should be deleted too.